Kernel/FileSystem/ISO9660FileSystem.h

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright (c) 2021, sin-ack <sin-ack@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/EnumBits.h>
+#include <AK/HashMap.h>
+#include <AK/NonnullRefPtr.h>
+#include <AK/StringView.h>
+#include <AK/Types.h>
+#include <Kernel/FileSystem/BlockBasedFileSystem.h>
+#include <Kernel/FileSystem/Inode.h>
+#include <Kernel/KBuffer.h>
+#include <Kernel/KResult.h>
+
+namespace Kernel {
+
+namespace ISO {
+
+template<typename T>
+struct [[gnu::packed]] LittleAndBigEndian {
+    T little;
+    T big;
+};
+
+// 8.4.26.1 Date and Time Format
```

Comment:
Is the ISO spec free? Can we link to the spec document in a comment here?

Reply:
ECMA 119 is free and describes the same format as ISO 9660. It's linked at the top comment too: https://www.ecma-international.org/wp-content/uploads/ECMA-119_4th_edition_june_2019.pdf

Reply:
Oh I see what you mean. Ooops.